apps/web/src/components/PostEditor/PostEditor.js

```diff
@@ -342,9 +370,21 @@ function PostEditor ({
     }
   }, [currentPost])
 
-  const handleDetailsChange = useCallback((event) => {
+  /**
+   * Handles changes to the post content in the editor
+   * @param {string} html - The HTML content from the editor
+   */
+  const handleDetailsChange = useCallback((html) => {
+    // Early return if no content is provided
+    if (!html) return
```

Comment:
hmm, i dont think is right, you want to be able to remove all the content

Reply:
though maybe it would still be an empty string which is truthy, so probably this is ok?